packages/app/features/Send/InformationInput/components/TransactionFee/TokenFeeDropDown.tsx

```diff
@@ -0,0 +1,77 @@
+import type { FC } from 'react';
+import { Image, StyleSheet } from 'react-native';
+import type { Token } from '@walless/core';
+import { Hoverable, modalActions, Text, View } from '@walless/gui';
+import type { TokenDocument } from '@walless/store';
+
+interface Props {
+	tokens: TokenDocument[];
+	onSelect: (token: Token) => void;
+	selectedToken?: Token;
+}
+
+const TokenFeeDropDown: FC<Props> = ({ tokens, onSelect, selectedToken }) => {
+	const handleSelectToken = (token: TokenDocument) => {
+		onSelect(token);
+		modalActions.destroy('NetworkFee');
+	};
+
+	const selectedStyle = {
+		backgroundColor: '#56667466',
+	};
```

Comment:
if style do not use any information from component (i.e props, state), it can be moved to `styles`